tests/models/falcon_mamba/test_modeling_falcon_mamba.py

```diff
@@ -524,3 +524,31 @@ def test_batched_generation(self):
         out = tok.batch_decode(out, skip_special_tokens=True)
 
         self.assertListEqual(out, EXPECTED_OUTPUT)
+
+    def test_training_kernel(self):
```

Comment:
Should be a slow test, no? Even with little tokens, I'd assume that it's still quite expensive.

Reply:
Indeed ! there is a decorator in the class `FalconMambaIntegrationTests` that should be propagated to all tests including this one so all should be good :)

Reply:
Ah, that makes sense! I overlooked that ^^